extensions/extension_mgr.go

```diff
@@ -20,31 +21,46 @@ type AdvancedExtension = extensions.AdvancedExtension
 const SubstraitDefaultURIPrefix = "https://github.com/substrait-io/substrait/blob/main/extensions/"
 
 // DefaultCollection is loaded with the default Substrait extension
-// definitions with the exception of decimal arithmetic. Decimal arithmetic
-// functions are missing as the complex return type expressions are not
-// yet implemented.
+// definitions.
+// Parser needs to enhanced until than below function files are not processed
+// 1. functions_arithmetic_decimal.yaml (need to parse complex return type)
```

Comment:
let's remove the specific names here. seems like overkill.

Reply:
I will fix it in next PR

Reply:
fixed in #61